data-manipulation/compression/extract-files-from-cabinet.yml

```diff
@@ -0,0 +1,21 @@
+rule:
+  meta:
+    name: extract files from Cabinet
```

Comment:
Thoughts on changing this to `extract Cabinet on Windows` to make it more clear?

Reply:
Thanks! Renamed the rule in https://github.com/mandiant/capa-rules/pull/808/commits/ac09516cca968e54d6b5764405337cba3f290c21